OpenRA.Mods.Common/Traits/Buildings/DockHost.cs

```diff
@@ -0,0 +1,169 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+using System;
+using System.Collections.Generic;
+using OpenRA.Activities;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	public sealed class DockType { DockType() { } }
```

Comment:
What is this for?

Reply:
It's the DockType class, implemented in the same way we do for all `...Types`